frontend/src/__tests__/components/filters/Filters.test.js

```diff
@@ -99,6 +100,33 @@ describe('Filters tests', () => {
     );
   });
 
+  it('renders active filters caption for filters without parameters', () => {
+    const updateDocListListener = jest.fn();
+    const dummyProps = createInitialProps(undefined, {
+      updateDocList: updateDocListListener,
+    });
+    const initialState = createStore({
+      windowHandler: {
+        allowShortcut: true,
+        modal: {
+          visible: false,
+        },
+      },
+      filters: filtersStoreFour,
+    });
+
+    const store = mockStore(initialState);
+    const wrapper = shallow(
+      <Provider store={store}>
+        <Filters {...dummyProps} />
+      </Provider>
+    );
+
+    wrapper.find(
+      '.filter-wrapper button[title="Abrechnung_offen_normal"]'
+    );
```

Comment:
don't see where is the assumpion/expectation.... or this test is not done yet...

Reply:
It works the same way as `expect(html).toContain('whatever')` but uses selector instead of string query, so it's supposed to be more efficient. Will fail the same way.